{{cookiecutter.project_name}}/mkdocs.yml

```diff
@@ -2,20 +2,37 @@
 site_name: "{{cookiecutter.project_name}}"
 theme:
   name: material
-  # palette:
-  #  scheme: slate
-  #  primary: cyan
+  palette:
+    # https://github.com/facelessuser/mkdocs_pymdownx_material_extras
+    scheme: dracula
+    primary: deep purple
+    accent: deep purple
```

Comment:
I'm not sure why we're changing the palette here. I don't love the original scheme, and prefer dark mode myself, but I don't think we should make a change like this without consulting our community.

Reply:
I have updated, thanks for reviewing.